docs/index.rst

```diff
@@ -83,6 +80,7 @@ Contents
    dates-and-times
    scylla-cloud
    scylla-cloud-serverless
+   column-encryption
    faq
 
```

Comment:
Sorry I didn't mark it explicitly, I thought you'll notice that. This one should be removed too.

Reply:
done